src/TTextEdit.h

```diff
@@ -156,12 +160,12 @@ private slots:
     TBuffer* mpBuffer;
     TConsole* mpConsole;
     QPointer<Host> mpHost;
-    QScrollBar* mpScrollBar;
     // screen height in characters
     int mScreenHeight;
     // currently viewed screen area
     QPixmap mScreenMap;
     int mScreenWidth;
+    int mScreenOffset = 0;
```

Comment:
You don't need the `= 0` if you have it properly in the initialiser list. This is a later C++xx addition to the standards which does not cause problems compared to the initialiser list technique (which can cause problems if the order is wrong!) - OTOH, some of us are more used to the earlier way which can be used if there are multiple constructors for a class (not sure but I guess an initialiser list entry will override one here...!)